gallery/tutorials/data/ctffind4/diagnostic_output.txt

```diff
@@ -1,6 +1,6 @@
 # Output from CTFFind version 4.1.14, run on 2023-02-02 15:13:07
 # Input file: test_img.mrc ; Number of micrographs: 1
-# Pixel size: 1.000 Angstroms ; acceleration voltage: 200.0 keV ; spherical aberration: 2.26 mm ; amplitude contrast: 0.07
-# Box size: 1024 pixels ; min. res.: 30.0 Angstroms ; max. res.: 5.0 Angstroms ; min. def.: 5000.0 um; max. def. 50000.0 um
-# Columns: #1 - micrograph number; #2 - defocus 1 [Angstroms]; #3 - defocus 2; #4 - azimuth of astigmatism; #5 - additional phase shift [radians]; #6 - cross correlation; #7 - spacing (in Angstroms) up to which CTF rings were fit successfully
+# Pixel size: 1.000 angstroms ; acceleration voltage: 200.0 keV ; spherical aberration: 2.26 mm ; amplitude contrast: 0.07
```

Comment:
This is the actual output of the CTFFIND4 tool, so it should probably stay the same. (Same below for `input.txt`.)

Reply:
Thanks. This also needs to be changed for `input.txt`.